js/audio/README.md

```diff
@@ -2,14 +2,65 @@
 
 ```svelte
```

Comment:
I think this needs to be `html`. Its not showing up in the page.

Reply:
yup i copied from #6136 to test but we'll fix it there